### ISSUE TITLE
Can not input Chinese characters on Mac version

### ISSUE THREAD

**Issue by yxw** [aardappel/treesheets#24]
When switch the input method to Chinese Pinyin, the keyed in characters in TreeSheets are still ASCII characters. This happens on Mac (OSX Yosemite version 10.10.1).

**Comment by WangWei90** [comment]
https://forums.wxwidgets.org/viewtopic.php?f=1&t=45553

I encounter this on the windows10, and maybe this will help, I don't know.

![image](https://user-images.githubusercontent.com/20700315/163740799-7aff92e0-7faa-438a-88ce-7a56d487f8d6.png)

**Comment by WangWei90** [comment]
Hello. I find that the search box and replace box are working well here.

![image](https://user-images.githubusercontent.com/20700315/164177327-451c46c3-e156-434a-901c-4052f19d2772.png)

![image](https://user-images.githubusercontent.com/20700315/164177342-cf10309b-db52-41e8-b99c-faedc3d7b42d.png)

**Comment by aardappel** [comment]
Yes, the main editing is a custom editing method, that probably doesn't support IME, unlike the search box which is standard Windows.
